agent/mapcall/src/main/resources/default-replacements.txt

```diff
@@ -0,0 +1,35 @@
+// Default replacement file
+//
+// Notes:
+// 1. This replacement file only uses package replacement.  It requires that all desired
+//    replacements exist in the replacement (e.g., second) package, and that there is a
+//    corresponding method in the original package.
+// 2. Experiments showed that creating a blanket null/zero-replacement of methods/constructors for
```

Comment:
Drop irrelevant information such as "experiments showed".  (The experiments weren't necessary, anyway.)
Just give the facts.  Also explain why; this text is vague.

Furthermore, this is important information that should go in the manual rather than here where users are unlikely to find it.  It's useful advice to people who are writing their own replacements.